### PR TITLE
fixed "EPERM" error for Windows user.

### PR DESCRIPTION
**Pull request by Fchen48**
If you will scan on Windows you will receive the following error: 
```js
Error: EPERM: operation not permitted, scandir '\documents and settings'
```
This pr will fix this in a simple way. 

`npm test` doesn't work for me, it will fail directly after fork, I think the size will calculate differently on Windows. 
Ok, CI passed.

### CODE REVIEW

File: lib/directory-tree.js [hunk 1]
```diff
@@ -12,9 +12,8 @@ function safeReadDirSync (path) {
 	try {
 		dirData = FS.readdirSync(path);
 	} catch(ex) {
-		if (ex.code == "EACCES")
-			//User does not have permissions, ignore directory
-			return null;
+		//User does not have permissions, ignore directory
+		if (ex.code == "EACCES" || ex.code == "EPERM") return null;
```
Comment (mihneadb) [inline lib/directory-tree.js]: Please keep the coding style as-is (`return null`) on the separate line.

Comment (Fchen48) [reply lib/directory-tree.js]: I changed it to the previous coding style

**Comment by TomoyukiAota** [comment]
I encountered EPERM when accessing `C:\Users\MyUserName\Documents\My Music`.
`My Music`, `My Pictures`, `My Videos` in `Documents` causes EPERM error. The reason is describe [here](https://www.experts-exchange.com/questions/27932035/access-denied-to-My-Pictures.html).

Currently, if any of files/directoies emits EPERM, this npm module just throws an error and fails to return the directory tree object. With this PR, it ignores such files/directories, which is a more useful behavior. (Also, if it can let users handle such errors, that's great, but the minimum viable behavior is just to ignore instead of throwing an error.)

@mihneadb 
Is this PR going to be merged?

**Comment by Fchen48** [comment]
@mihneadb Any problems with the PR?

**Comment by mihneadb** [comment]
@Fchen48 sorry about the lag. Merging & releasing now. Thanks!

**Pull request merged**